x11-libs/libfm-qt/libfm-qt-0.15.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake xdg-utils
+
+DESCRIPTION="Qt port of libfm, a library providing components to build desktop file managers"
+HOMEPAGE="https://lxqt.org/"
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxqt/${PN}.git"
+else
+	SRC_URI="
+		https://downloads.lxqt.org/downloads/${PN}/${PV}/${P}.tar.xz
+		https://github.com/lxqt/${PN}/releases/download/${PV}/${P}.tar.xz
+	"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+fi
+
+LICENSE="BSD GPL-2+ LGPL-2.1+"
+SLOT="0/7"
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+	>=dev-util/lxqt-build-tools-0.7.0
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-libs/glib:2
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5=
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	>=lxde-base/menu-cache-1.1.0:=
+	media-libs/libexif:=
```

Comment:
This doesn't seem to have a subslot, so the `:=` here seems useless.

Reply:
I was actually wondering about that too.  I looked at the devmanual, and it says that "If an ebuild does not explicitly declare a sub-slot, the regular slot is used as the value of the sub-slot by default.", so I figured it was put there to force a rebuild in any case.

(Did that package maybe have a subslot in the past?)

During all this I noticed that some of the LXQt subslots had not been updated since 0.13 maybe... ooops.